specutils/tests/test_fitting.py

```diff
@@ -595,3 +595,55 @@ def test_spectrum_from_model():
                               1.5319218, 1.06886794, 0.71101768, 0.45092638, 0.27264641])
 
     assert np.allclose(spectrum_gaussian.flux.value[::10], flux_expected, atol=1e-5)
+
+
+def test_masking():
+    """
+    Test fitting spectra with masks
+    """
+    x, y = double_peak()
```

Comment:
avoid single-character variable names.... (my preference, not an official style recommendation, afaik)

Reply:
I think it's also in several of the default linters, so it's quasi-official... regardless, good point will fix